packages/app/src/store/specs-store.ts

```diff
@@ -1,8 +1,10 @@
 import type { SpecFile } from '@packages/types/src'
 import { defineStore } from 'pinia'
 
+export type SpecWithFilter = SpecFile & { testFilter?: Cypress.Spec['testFilter'] }
+
 export interface SpecState {
-  activeSpec: SpecFile | null | undefined
+  activeSpec: SpecWithFilter | undefined | null
```

Comment:
```suggestion
  activeSpec?: SpecWithFilter
```

Not sure if this is valid, but if it is, it would be more concise.

Reply:
I've reverted the changes made to this file.